src/mixins/bcol-mixin.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Vue } from 'vue-property-decorator'
+import { AlertMessageIF } from '@/interfaces'
+import axios from '@/axios-auth'
+
+@Component({})
+export default class BcolMixin extends Vue {
+  async getErrorObj (url, errCode) {
```

Comment:
probably don't need the url parameter, you are already fetching it from session storage.

Reply:
removed

Reply:
Since Ody is away this week, I'll say this looks satisfactorily handled and am closing this conversation.